.eslintrc.json

```diff
@@ -21,8 +21,8 @@
   "parserOptions": {
     "ecmaVersion": "latest",
     "sourceType": "module",
-    // "project": ["tsconfig.base.json"]
-    "project": true
+    "project": ["tsconfig.base.json"]
```

Comment:
Please change this file.

Reply:
This change is required because I changed the "global-wallets-env.d.ts" file. As a result of the change in that file, it becomes part of the staged changes, leading to an encountered error during ESLint's linting process. Consequently, I had to modify this line in the ESLint config file to address the issue.